internal/commands/user.go

```diff
@@ -35,3 +37,102 @@ func getSlackUserInfo(
 
 	return &user, err
 }
+
+func getUsersIDsInConversation(
+	ctx context.Context,
+	client bot.Client,
+	logger log.Logger,
+	channelID string,
+) (*[]string, error) {
+	logger.Info(
+		ctx,
+		"command/user.getUsersInConversation",
+		log.NewValue("params", channelID),
+	)
+
+	params := slack.GetUsersInConversationParameters{
+		ChannelID: channelID,
+	}
+
+	var members []string
+
+	for {
+		list, cursor, err := client.GetUsersInConversationContext(ctx, &params)
+		if err != nil {
+			logger.Error(
+				ctx,
+				"command/user.getUsersIDsInConversation",
+				log.NewValue("channelID", channelID),
+				log.NewValue("error", err),
+			)
+			return nil, err
+		}
+		members = append(members, list...)
+		if cursor == "" {
+			break
+		} else {
+			params.Cursor = cursor
+		}
+	}
+
+	return &members, nil
+}
+
+func getUsersInConversation(
+	ctx context.Context,
+	client bot.Client,
+	logger log.Logger,
+	channelID string,
+) (*[]model.User, error) {
+	var users []model.User
+
+	usersIDs, err := getUsersIDsInConversation(ctx, client, logger, channelID)
+	logger.Error(
+		ctx,
+		"command/user.getUsersInConversation",
+		log.NewValue("channelID", channelID),
+		log.NewValue("error", err),
+	)
```

Comment:
It is not a necessary test if there is an error? So you could step out of the function earlier.
Like that, the code will keep running to the next function then it could fail when you look into the usersIDs
Maybe: 

```
if err != nil {
	logger.Error(
		ctx,
		"command/user.getUsersInConversation",
		log.NewValue("channelID", channelID),
		log.NewValue("error", err),
	)
return nil, err
}
```

Reply:
You are right! I totally forgot about this.

Thanks for the remind!

Reply:
Fixed